modules/ws-client/src/main/scala/trading/client/WS.scala

```diff
@@ -0,0 +1,89 @@
+package trading.client
```

Comment:
This looks good! Shall I raise an issue on the repo and point it this way?

Reply:
Sure! I'd be happy to contribute something back to Tyrian :)